cdn/dovetail3-cdn.yml

```diff
@@ -358,7 +358,7 @@ Resources:
       Namespace: AWS/CloudFront
       Period: 60
       Statistic: Average
-      Threshold: 0.5
```

Comment:
Seemed a bit weird to look at these fractional percentages.  These will probably need updating as we get more traffic moved over anyways.